File: introduction/ITP1_1_D-Watch.cs
using System;

public class Program {
  public static void Main() {
    int time = Int32.Parse(Console.ReadLine());
    int hour = time / 3600;
    int minutes = (time / 60) % 60;
    int seconds = time % 60;
    Console.WriteLine("{0}:{1}:{2}", hour, minutes, seconds);
  }
}

File: introduction/ITP1_1_D-Watch.js
process.stdin.resume();
process.stdin.setEncoding('utf8');
process.stdin.on('data', function (chunk) {
  var time = parseInt(chunk);
  var hour = parseInt(time / 3600);
  var minutes = parseInt((time / 60) % 60);
  var seconds = parseInt(time % 60);
  console.log(hour + ':' + minutes + ':' + seconds);
});
